recipes/m2crypto/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Set the OPENSSL prefix so it doesn't link to /usr/lib/libssl
+OPENSSL_PATH=$PREFIX $PYTHON setup.py install
```

Comment:
If we are using `setuptools`, we need to add ` --single-version-externally-managed --record record.txt`.